tests/tools/importexport/migration/test_v02_to_v03.py

```diff
@@ -137,7 +79,7 @@ def test_compare_migration_with_aiida_made(self):
         metadata_v3.pop('aiida_version')
         self.assertDictEqual(metadata_v2, metadata_v3)
 
-        self.maxDiff = None
+        self.maxDiff = None  # pylint: disable=invalid-name
```

Comment:
Since it's set in the `setUpClass` of the parent class, it shouldn't need to be set again.
```suggestion
```